solidity/dashboard/src/css/typography.less

```diff
@@ -17,71 +17,81 @@ body {
   font-family: 'Work Sans', sans-serif;
   font-weight: normal;
   font-size: 1rem;
+  line-height: 1.5rem;
   color: @color-black;
-  line-height: 1.44;
+  font-feature-settings: 'tnum' on, 'lnum' on, 'zero' on;
 
   @media @tablet-up {
     line-height: 1.79;
   }
 }
 
+h1, .h1,
+h2, .h2,
+h3, .h3,
+h4, .h4,
+h5, .h5,
+h6, .h6 {
+  font-family: 'Work Sans', sans-serif;
+  font-feature-settings: 'tnum' on, 'lnum' on, 'zero' on;
```

Comment:
Are these two necessary, given inheritance from `body`? Are there cases where the headers need to have the font but intervening elements have changed it?

Reply:
pls see c3d1712